src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,38 @@
+.Navbar {
+  width: 100%;
+  height: 50px;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  top: 0;
+  font-size: 16px;
+  background-color: var(--color-white);
+  
+  &__left {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+  
+  &__right {
+    flex: 3;
+    gap: 15px;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    
+    .navbar__links {
+      font-family: var(--inter);
+      color: var(--color-bunting);
+    }
+    
+    .navbar__btn {
+      padding: 16px;
+      color: var(--color-blue-ribbon);
+      border-radius: 8px;
+      outline: 2px solid;
+      outline-color: var(--color-blue-ribbon);
+      font-family: var(--inter);
+    }
+  }
+}
```

Comment:
we also need to resize the image we imported, I believe
e.g. 
.Navbar__icon img {
  width: 150px;
}